pkgs/top-level/python-packages.nix

```diff
@@ -30460,28 +30463,46 @@ in {
     };
   };
 
-  # tensorflow is built from a downloaded wheel, because the upstream
-  # project's build system is an arcane beast based on
-  # bazel. Untangling it and building the wheel from source is an open
-  # problem.
+  # TensorFlow is built from a downloaded wheel, because the upstream
+  # project's build system is an arcane beast based on bazel. Untangling
+  # it and building the wheel from source is an open problem.
 
-  tensorflow = self.tensorflowNoGpuSupport;
+  tensorflow = self.tensorflowWithoutGpu;
 
-  tensorflowNoGpuSupport = buildPythonPackage rec {
+  tensorflowWithoutGpu = buildPythonPackage rec {
```

Comment:
Is this with generic GPU support, or specifically Cuda? If I am correct it is Cuda only, and therefore the name should reflect that: `tensorflowWithoutCuda` and `tensorflowWithCuda`